finality-aleph/src/data_io/data_store.rs

```diff
@@ -309,6 +309,27 @@ where
                             debug!(target: "aleph-data-store", "Requesting a stale block {:?} after it has been missing for {:?} secs.", block, time_waiting.as_secs());
```

Comment:
nit: wrap these long lines in `run_maintenance`

Reply:
Ok, done -- you mean something like
```
"bla bla bla \
bla bla bla"
```
right?

Reply:
yeah, or you can turn on (for a while) an unstable option of `rustfmt` for formatting inside macro

Reply:
Ok, went for the manual slicing.